src/hooks/use-debounce.tsx

```diff
@@ -0,0 +1,25 @@
+import { useRef, useEffect } from 'react'
+
+type TimerType = ReturnType<typeof setTimeout>;
+type CallbackType<T> = (...args: T[]) => void;
+
+export const useDebounce = <T,>(callback: CallbackType<T>, delay = 500): CallbackType<T> => {
```

Comment:
Typo, delete comma

Reply:
I can`t because if I delete vite thinks that is jsx and throws an error, it can be fixed by changing the extension to ts from tsx but then vite can`t compile it. We need to fix vite config in other issue

Reply:
Ok. I gotcha, so now we have a new issue.